src/main/java/edu/carrollcc/cis232/Question1.java

```diff
@@ -2,9 +2,24 @@
 
 public class Question1 {
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws IllegalArgumentException {
 
 	}
 
-	//Write static maxElement method here
-}
+	// Write static maxElement method here
+	public static int maxElement(int[] numbers) {
+		if (numbers == null || numbers.length == 0) {
+			throw new IllegalArgumentException("Error");
+		}
+		return maxElement(numbers, 0);
```

Comment:
The way you are doing recursion, you should be starting at `i` equal to the length of the array minus 1.  That would make the rest of the code work